src/dndbeyond/base/utils.js

```diff
@@ -266,7 +266,8 @@ async function buildAttackRoll(character, attack_source, name, description, prop
 
         // TODO: refactor into a method and remove it from build attack roll 
         if (character.hasClass("Rogue") && character.hasClassFeature("Sneak Attack 2024") && 
-            character.getSetting("rogue-sneak-attack", false) && 
+            (character.getSetting("rogue-sneak-attack", false) || 
+            name.includes("Sneak Attack") && !character.getSetting("rogue-sneak-attack", false) && character.getSetting("rogue-cunning-strike", false)) && 
```

Comment:
Very confusing statement because of the mix of `||` and `&&` without separating them in parenthesis (I know there's a priority order on the AND and OR operators, but without grouping them, it's hard to wrap my head around the order.
You also don't need to check for `!rogue-sneak-attack` since you're already in an `||` from the truthy check, so if it checks for the `name.includes`, we already know it's disabled.